documentation/public/user-guide/08-examples/01-list-functions-in-contract/examples/test-list-functions.test.mts

```diff
@@ -0,0 +1,14 @@
+import assert from "node:assert";
+import { listFunctionsInContract } from "./list-functions-in-contract.mjs";
+import buildSampleCompilationUnit from "../../common/sample-contract.mjs";
+
+test("list functions by contract name", async () => {
+  const unit = await buildSampleCompilationUnit();
+
+  const functions = listFunctionsInContract(unit, "Counter");
+  assert(functions);
+  assert.strictEqual(functions.length, 2);
```

Comment:
I think the `deepEqual()` below would implicitly check the functions count as well?